ui/src/components/errorBanner.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default function ErrorBanner({ error, id }) {
+  return error ? (
+    <div id={id}>
```

Comment:
The thought was to make these findable in Selenium tests but I didn't finish the work. I'm going to drop `id` for now, since I think the long-term solution will be to replace this code with react-alert.